packages/admin/admin-theme/src/componentsTheme/MuiAlert.tsx

```diff
@@ -19,9 +19,8 @@ export const getMuiAlert: GetMuiComponentTheme<"MuiAlert"> = (component, { palet
     },
     styleOverrides: mergeOverrideStyles<"MuiAlert">(component?.styleOverrides, {
         root: {
-            width: 400,
+            maxWidth: 400,
```

Comment:
I'd remove this altogether, sizing should be done by the enclosing container. @jamesricky what do you think?

Reply:
I agree, the width should be defined by the parent's width.